src/Core/Content/Product/SalesChannel/SalesChannelProductEntity.php

```diff
@@ -38,6 +39,11 @@ class SalesChannelProductEntity extends ProductEntity
 
     protected ?CheapestPriceContainer $cheapestPriceContainer = null;
 
+    protected ?ConvertedUnitSet $measurements = null;
+
+    /**
+     * @deprecated tag:v6.8.0 - Will be removed in 6.8.0. use $measurements instead.
```

Comment:
6.7.1.0 is not released yet, so if this change will be part of it, no need for deprecation. 

@keulinho @pweyck

Reply:
okay thought it was too late 🙈